.tekton/multicluster-operators-subscription-acm-214-pull-request.yaml

```diff
@@ -117,6 +123,11 @@ spec:
         set of values is determined by the configuration of the multi-platform-controller.
       name: build-platforms
       type: array
+    - default: "true"
+      description: |
+        Check symlinks in the repo. If they're pointing outside of the repo, the build will fail.
+      name: enableSymlinkCheck
+      type: string
```

Comment:
This should be removed. The param to ignore symlinks is task-specific rather than pipeline-specific, so the one below this in `clone-repository` is all that should be needed.
```suggestion
```